### DISCUSSION TITLE
New shortcut for Base Environment and/or mask Private environment variables on dialog open

### DISCUSSION THREAD

**Discussion by SohamJ** [Kong/insomnia#4080] [Ideas]
Using the Command + E shortcut opens up the default sub-environment. If the default sub-environment is a Private environment, using the shortcut exposes private environment variables like passwords and API tokens. 

Instead, there could be a shortcut that opens the base environment (my requirement). Optionally, the private environment can mask the JSON values by default and have the option to `Show values` (could be configurable through global settings

**Comment by mroche89** [comment]
👍  I use the base env to store a lot of global state using template tags from different responses. It's an extra step to get here so a shortcut directly to base env (and folder environment for the current request) would be amazing
